lib/kube/proxy/forwarder_test.go

```diff
@@ -988,90 +978,13 @@ func mockAuthCtx(ctx context.Context, t *testing.T, kubeCluster string, isRemote
 	}
 }
 
-func TestNewClusterSessionLocal(t *testing.T) {
```

Comment:
These tests didn't test much. If you check, all the important validations were skipped in the `cmp.Diff`.
I removed them since in reality the test just checked if everything was populated and didn't validate the contents.